buildSrc/src/main/groovy/nva.cristin.service.core.java-conventions.gradle

```diff
@@ -54,6 +54,30 @@ task karateTest(type: Test) {
     outputs.upToDateWhen { false }
 }
 
+task createTestUsers(type: Test) {
+    useJUnitPlatform {
+        includeTags('createTestUsers')
+    }
+
+    environment "COGNITO_USER_POOL_ID",  "eu-west-1_DNRmDPtxY"
+    environment "COGNITO_CLIENT_APP_ID",  "4qfhv3kl9qcr2knsfb8lhu1u40"
+    environment "TESTUSER_FEIDE_ID",  "karate-user-administrator@sikt.no"
+    environment "TESTUSER_PASSWORD", "AP@ssW00rd2Remember"
```

Comment:
Is this just a dummy password?

Reply:
Nope, it's the users password - couldn't get dynamic password generation to work, nor github secret